root/release/edit/tracklist.tt

```diff
@@ -66,6 +66,22 @@
         </label>
       </td>
     </tr>
+    <tr>
+    <td>
+      <label>
+      <input type="checkbox" name="use-custom-delimiter" data-bind="checked: useCustomDelimiter" />
+        [%= l('Use custom artist delimeter') %]
+      </label>
+    </td>
+      <td>
+        <label data-bind="visible: useCustomDelimiter">
+          [%= l('Custom delimiter') %]
+          <input type="text" name="custom-delimiter" data-bind="value: customDelimiter" />
+          (<a href="#" data-bind="click: toggleDelimiterHelp">[% l('help') %]</a>)
+          <div data-bind="visible: delimiterHelpVisible"><i>{l('Enter the delimiter that separates the track name from the artist name (i.e. "," or "\t").')}</i></div>
```

Comment:
JSX syntax (`{}`) in TT?

Reply:
I don't see any reason not to allow for any RegExp here.

Reply:
Any regexp seems good to me too.

Reply:
We can probably just link to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp then.
```suggestion
          <div data-bind="visible: delimiterHelpVisible"><i>[% l('Enter the delimiter that separates the track name from the artist name (any valid {url|regular expression} is accepted).', { url => 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp' }) %]</i></div>
```

Reply:
Great, thanks guys.

Reply:
Or https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#Writing_a_regular_expression_pattern might be friendlier

Reply:
Or https://developer.mozilla.org/docs/Web/JavaScript/Guide/Regular_Expressions for localized version.